dev/manual_tests/lib/raw_keyboard.dart

```diff
@@ -105,7 +105,7 @@ class _HardwareKeyDemoState extends State<RawKeyboardDemo> {
           } else if (data is RawKeyEventDataLinux) {
             dataText.add(Text('keyCode: ${data.keyCode} (${_asHex(data.keyCode)})'));
             dataText.add(Text('scanCode: ${data.scanCode}'));
-            dataText.add(Text('codePoint: ${data.codePoint}'));
+            dataText.add(Text('charactersIgnoringModifiers: ${data.charactersIgnoringModifiers}'));
```

Comment:
Does this also work for all the other platforms we support? I remember @gspencergoog saying that not all data is available everywhere...

Reply:
(or should we provide both properties?)

Reply:
Each platform has their own RawKeyEventData with different properties, so this change wouldn't affect them. There's no need to send both since codePoint is just the decimal representation of the characters (which we were converting to string anyway).